clients/drcachesim/tests/burst_syscall_inject.cpp

```diff
@@ -112,7 +113,10 @@ do_some_syscalls()
                   KERNEL_SIGSET_SIZE);
     assert(res != 0 && errno == EFAULT);
 
-    std::cerr << "Done with system calls\n";
+    // XXX i#6490: Printing to stderr here causes flakiness in the test
```

Comment:
(This seems really familiar...oh ok I see #6490 you cited here is not the same as this PR's #6495 and yes I think that is what I was thinking of.)

Reply:
Yup. I think it's likely the same thing affecting these burst tests.